addons/csi/openstack/snapshot-crds.yaml

```diff
@@ -13,11 +13,7 @@
 # limitations under the License.
 
 # Sourced from
-# https://github.com/kubernetes-csi/external-snapshotter/tree/release-4.2/client/config/crd
-
-{{ if .Cluster.Features.Has "externalCloudProvider" }}
-{{ if eq .Cluster.CloudProviderName "openstack" }}
```

Comment:
I think we need this

Reply:
ah damn it, that was unintentional. Thanks!